src/module_2/EDA_1.md

```diff
@@ -0,0 +1,3314 @@
+print(f"Número de carros abandonados que también se compraron: {total_abandonos_comprados}")
+
+# Resumen final
+print("Resumen del análisis de carros abandonados:")
+print(f"Número total de carros abandonados: {total_abandonos}")
+print(f"Número de carros abandonados que también se compraron: {total_abandonos_comprados}")
+
+
+```
+
+    Número total de carros abandonados: 5457
+    Items más abandonados (por variant_id):
+                variant_id  cantidad_abandonos
+    1240  [34081589887108]                 158
+    4586  [39709997760644]                  39
+    1954  [34284950519940]                  24
+    363   [33667268083844]                  23
+    1419  [34137590366340]                  21
+    1661  [34284949766276]                  19
+    576   [33824368033924]                  17
+    4645  [39709997858948]                  14
+    3061  [34543001370756]                  12
+    1796  [34284950356100]                  11
+    Usuarios que abandonan carros con mayor frecuencia:
+       user_id  cantidad_abandonos
+    0      128                5457
+    Número de carros abandonados que también se compraron: 0
+    Resumen del análisis de carros abandonados:
+    Número total de carros abandonados: 5457
+    Número de carros abandonados que también se compraron: 0
+
+
+
+```python
+#Revision_de_distribución_de_user_id
+
+# Verificar la cantidad de usuarios únicos en el dataset
+usuarios_unicos = abandoned_cart['user_id'].nunique()
+print(f"Número de usuarios únicos: {usuarios_unicos}")
+
+# Mostrar los primeros 10 valores únicos de 'user_id' para verificar si están duplicados
+print(abandoned_cart['user_id'].unique()[:10])
+
+# Contar la frecuencia de cada 'user_id' para ver la distribución
+distribucion_usuarios = abandoned_cart['user_id'].value_counts()
+print(distribucion_usuarios.head(10))
+"Esto es inusual y sugiere que los datos pueden no estar bien representado"
+
+
+
+```
+
+    Número de usuarios únicos: 1
+    [128]
+    user_id
+    128    5457
+    Name: count, dtype: int64
+
+
+
+
+
+    'Esto es inusual y sugiere que los datos pueden no estar bien representado'
+
+
+
+
+```python
+#1. Verificación de tipos de datos
+#variant_id no es del tipo correcto
+
+# Verificar tipos de datos en abandoned_cart
+print(abandoned_cart.dtypes)
+
+# Verificar los tipos de los valores en variant_id
+print(abandoned_cart['variant_id'].apply(type).value_counts())
+# Asegurarte de que 'variant_id' es un string
+abandoned_cart['variant_id'] = abandoned_cart['variant_id'].astype(str)
+
+
+```
+
+    id                     int64
+    user_id                int64
+    created_at    datetime64[us]
+    variant_id            object
+    dtype: object
+    variant_id
+    <class 'numpy.ndarray'>    5457
+    Name: count, dtype: int64
+
+
+
+```python
+#Inventory_Analisys
+import pandas as pd
+
+# Define la ruta donde está el archivo Parquet
+data_path = '/home/eduardo_1999/projects/zrive-ds/projects/zrive-ds/sampled-datasets/'
```

Comment:
Why loadding the data here and overwriting this variable again? Data loading was intended to be made on first cell...